frontend/docs/features/components/dependency-analysis.md

```
@@ -0,0 +1,48 @@
+npm run analyze:metrics   # Ca(팬인)/Ce(팬아웃)/I%(instability) 메트릭 표
+npm run analyze:circular  # 순환참조 탐지 (madge)
+npm run analyze:graph     # SVG 그래프 생성 (graphviz 필요)
+```
+
+`analyze:metrics`는 `depcruise src --focus "^src/components"`로 전체 그래프를 크롤링한 뒤 components만 필터링한다. `src/components`만 크롤링하면 `pages`에서 들어오는 import를 보지 못해 Ca가 전부 0으로 나온다.
+
+## 발견 사례: SearchBox 위치 오분류
+
+첫 실행에서 `common-no-pages` 규칙이 다음 위반을 잡았다.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)